english/time-machine/time-machine.md

```diff
@@ -4,3 +4,10 @@ There is no way back to the current time.
 So think carefully before you choose.
 
 Which part of the world, and what time you want to travel? And why?
+
+[You decide to travel to 2135 and say goodbye to the moonmen](https://www.youtube.com/watch?v=TgqiSBxvdws)
+
+[You travel to 2017 and meet with an Apple Engineer](https://www.youtube.com/watch?v=-XSC_UG5_kU)
+
+[You turn back](english/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.